contrib/packs/actions/load.yaml

```diff
@@ -7,8 +7,8 @@
   parameters:
     register:
       type: "string"
-      default: "actions,aliases,sensors"
-      description: "Possible options are all, sensors, actions, rules, aliases."
+      default: "actions,aliases,sensors,triggers"
```

Comment:
BTW, `triggers` are already in place here: 
https://github.com/StackStorm/st2/blob/8df8c5adaf8ec843702411fa8a00b029d898345c/contrib/packs/actions/install.meta.yaml#L19

so seems like it was unsynced someday.